src/pages/Admin/Dashboard.tsx

```diff
@@ -24,13 +23,9 @@ export default function Dashboard() {
     }
 
     return (
-        <IonContent style={{ '--background': 'white' }}>
-            <InformationBlock>
-                {t('dashboard.welcome')}
-                <Button onClick={() => startPremiumVideoCall()}>
-                    Start Call
-                </Button>
-            </InformationBlock>
-        </IonContent>
+        <InformationBlock>
+            {t('dashboard.welcome')}
+            <Button onClick={() => startPremiumVideoCall()}>Start Call</Button>
```

Comment:
```suggestion
            <Button onClick={startPremiumVideoCall}>Start Call</Button>
```

Reply:
only this suggestion. It's ok to me

Reply:
thanks man. Gonna apply that in the future, but this part of code will be removed by the merge of PR on admin dashboard